src/brs/props/Props.java

```diff
@@ -128,6 +128,8 @@ public class Props {
   public static final Prop<Integer> BRS_BLOCK_CACHE_MB = new Prop<>("node.blockCacheMB", 40);
   public static final Prop<Integer> BRS_AT_PROCESSOR_CACHE_BLOCK_COUNT = new Prop<>("node.atProcessorCacheBlockCount", 1000);
 
+  public static final Prop<Integer> DB_INSERT_BATCH_MAX_SIZE = new Prop<>("DB.InsertBatchMaxSize", 1000);
```

Comment:
Why 1000, and not e.g. 10000?

Reply:
I run it with 2500 and it worked well .. maybe 10k is already a problem for SQLite .. so as it is a parameter i would set it lower and if we get the 1.2M tx more often we can inform user to adjust the parameter ... so i would give with 2.500 as default - not higher

Reply:
A big factor on this number could also be when syncing a node with a HDD or SSD

Reply:
I assume it is a default value which may be set to 1M bytes or so by SQLite and if the statement goes above it triggers the error.. 

Maximum Length Of An SQL Statement

The maximum number of bytes in the text of an SQL statement is limited to SQLITE_MAX_SQL_LENGTH which defaults to 1,000,000,000.


Found on https://www.sqlite.org/limits.html

--

Maybe 5k is a good value in the middle

Reply:
> A big factor on this number could also be when syncing a node with a HDD or SSD The specs of the `server/computer`
Also why I thought 1000 should work for most hardware.

See also my comment below with statistics.
Only 259 blocks are more than 1000 rows to insert, since genesis block !

1000 or 2500 would be fine, I think.

Reply:
:+1: